website_event_form/static/src/js/website_event_form.js

```diff
@@ -0,0 +1,79 @@
+odoo.define('website_event_form.EventProcessFormBtn', function(require) {
+	'use strict';
+
+	var publicWidget = require('web.public.widget');
+	var time = require('web.time');
+
+	publicWidget.registry.EventProcessFormBtn = publicWidget.Widget.extend({
+		selector: '#event_process_form',
+		events: {
+			'click #event_process_form_btn': '_onClickProcessForm'
+		},
+		start: function() {
+			let self = this;
+			return this._super.apply(this, arguments).then(function() {
+				$('#event_start_date').datepicker({
+					showOtherMonths: true,
+					dateFormat: "yy-mm-dd",
+				});
+				$('#event_end_date').datepicker({
+					showOtherMonths: true,
+					dateFormat: "yy-mm-dd",
+				});
+
+				// NB: Save photo file before we submit the form to avoid async
+				// processes.
+				document.getElementById('event_photo').addEventListener(
```

Comment:
@KKamaa the code is fine, but the photo does not appear,.. the object has id=event_photo,  but something is not working..